test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -173,6 +173,47 @@ TEST_F(CLASSNAME(TestSubscription, RMW_IMPLEMENTATION), create_with_bad_argument
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
 }
 
+TEST_F(CLASSNAME(TestSubscription, RMW_IMPLEMENTATION), create_with_unsupported_cft_options) {
+  constexpr char topic_name[] = "/test";
+  const rosidl_message_type_support_t * ts =
+    ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, BasicTypes);
+
+  rmw_subscription_options_t options = rmw_get_default_subscription_options();
+  auto allocator = rcutils_get_default_allocator();
+
+  rmw_subscription_content_filter_options_t * content_filter_options =
+    static_cast<rmw_subscription_content_filter_options_t *>(
+    allocator.allocate(
+      sizeof(rmw_subscription_content_filter_options_t), allocator.state));
```

Comment:
Does this need to be allocated? Seems like it could just be on the stack.

Reply:
Good eye. I'll update it even if the variable on the heap works well.